app/src/components/MachineGrid.tsx

```diff
@@ -24,13 +24,13 @@ const MachineGrid: React.FC<GridProps> = ({ allMachines, subscribedMachines, sho
                 <div className="responsive-width grid grid-cols-2 gap-5 m-auto pb-20">
                     {machines?.map(function (machine) {
                         return (
-                            <Link to={`/machine/${machine?.id}`} key={machine?.id}>
-                                <MachineContainer
-                                    name={machine?.name}
-                                    health={machine?.healthStatus}
-                                    image={machine?.image}
-                                />
-                            </Link>
+                            <MachineContainer
+                                key={machine.id}
+                                name={machine.name}
+                                health={machine.healthStatus}
+                                image={machine.image || "random"} // random is a placeholder right now, as not all machiens have images
```

Comment:
I think we can remove the || "random"} now, since all machines have images (and will have images)

Reply:
I've now removed it